public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -283,6 +283,7 @@ export function initDashboard(args: InitDashboardArgs): ThunkResult<void> {
     trackDashboardLoaded(dashboard, versionBeforeMigration);
 
     // yay we are done
+    dashboard.setOriginalDashboard(dashDTO.dashboard);
```

Comment:
I think this setter is unnecessary. This model is passed to DashboardModel constructor on L193 and can be used to store the private property.

Reply:
Good catch!